appserver/batch/glassfish-batch-connector/src/main/java/fish/payara/jbatch/persistence/rdbms/JBatchJDBCPersistenceManager.java

```diff
@@ -2539,19 +2477,40 @@ public void shutdown() throws BatchContainerServiceException {
 
 	protected Map<String, String> getSharedQueryMap(IBatchConfig batchConfig) throws SQLException {
 		queryStrings = new HashMap<>();
-		queryStrings.put(Q_SET_SCHEMA, "SET SCHEMA ?");
-		queryStrings.put(SELECT_CHECKPOINTDATA, "select id, obj from "
-				+ tableNames.get(CHECKPOINT_TABLE_KEY) + " where id = ?");
-		queryStrings.put(INSERT_CHECKPOINTDATA,
-				"insert into " + tableNames.get(CHECKPOINT_TABLE_KEY)
-						+ " values(?, ?)");
-		queryStrings.put(UPDATE_CHECKPOINTDATA,
-				"update " + tableNames.get(CHECKPOINT_TABLE_KEY)
-						+ " set obj = ? where id = ?");
+		
+		queryStrings.put(
+	        Q_SET_SCHEMA, 
```

Comment:
Is this spacing necessary? Surely these fit on fewer lines.
We're not sticking to 80 character lines remember